apps/api/src/server/applications/application/representations/status/status.mapper.js

```diff
@@ -0,0 +1,9 @@
+import { pick } from 'lodash-es';
+
+export const updateRepStatusRequestToRepository = (body) => {
+	return {
+		...pick(body, ['status', 'notes', 'invalidReason', 'referredTo']),
+		actionBy: body.updatedBy,
+		type: 'STATUS'
+	};
+};
```

Comment:
Great question, I generally want us to avoid tightly coupling the DB -> API -> FE.  It helps isolate the changes and makes it more maintainable. This field is not a great example but the  DB table is a generic Action table that supports status updates and redaction events. Here the status was 'updated by', the fact it gets stored in DB as actionBy is really implementation specific.